writers/mongodb/mongodb.go

```diff
@@ -0,0 +1,29 @@
+package mongodb
+
+import (
+	"gopkg.in/mgo.v2"
```

Comment:
This lib is unmaintained. Maybe this one https://github.com/mongodb/mongo-go-driver?